master/docs/manual/cfg-workers-openstack.rst

```diff
@@ -106,12 +106,14 @@ These are the same details set in either environment variables or passed as opti
 ``nova_args``
     (optional)
     A dict that will be appended to the arguments when creating a VM.
-    Buildbot uses the OpenStack Nova version 1.1 API by default (see client_version).
+    Buildbot uses the OpenStack Nova version 2 API by default (see client_version).
```

Comment:
At the moment, yes. It depends on the version of novaclient.

For a while it has automatically converted to using v2 of the API instead of v1.1. If you need to use the older API version, you still can by changing `client_version` combined with an older version of novaclient.